src/map/lua/luautils.cpp

```diff
@@ -562,11 +562,30 @@ namespace luautils
             return -1;
         }
 
-        lua_getglobal(LuaHandle, "SetRegionalConquestOverseers");
+        lua_getglobal(LuaHandle, "dsp");
         if (lua_isnil(LuaHandle, -1))
         {
             lua_pop(LuaHandle, 1);
-            ShowError("luautils::SetRegionalConquestOverseers: undefined procedure SetRegionalConquestOverseers\n");
+            ShowError("luautils::SetRegionalConquestOverseers: undefined global dsp\n");
+            return -1;
+        }
+
+        int TableIndex = 0;
+        TableIndex = lua_gettop(LuaHandle);
+        lua_getfield(LuaHandle,TableIndex,"conquest");
+        if (lua_isnil(LuaHandle, -1))
+        {
+            lua_pop(LuaHandle, 1);
+            ShowError("luautils::SetRegionalConquestOverseers: undefined field dsp.conquest\n");
+            return -1;
+        }
+
+        TableIndex = lua_gettop(LuaHandle);
+        lua_getfield(LuaHandle,TableIndex,"setRegionalConquestOverseers");
+        if (lua_isnil(LuaHandle, -1))
+        {
+            lua_pop(LuaHandle, 1);
+            ShowError("luautils::SetRegionalConquestOverseers: undefined procedure dsp.conquest.setRegionalConquestOverseers\n");
```

Comment:
i can't comment down far enough so i'll put it on this line, but I'm pretty sure you'll also need to pop 2 extras after the lua_pcall (ie. pop 3 if it fails instead of 1, and pop 2 if it succeeds).  Easy way to test: print lua_gettop before doing anything in this function, and then print it again after everything is done - since it pcall doesn't return any values, it should be the same before and after